lib/backpex/fields/has_many.ex

```diff
@@ -538,9 +538,10 @@ defmodule Backpex.Fields.HasMany do
   defp assign_form_errors(socket) do
     %{assigns: %{form: form, name: name, field_options: field_options}} = socket
 
+    errors = if Phoenix.Component.used_input?(form[name]), do: form[name].errors, else: []
```

Comment:
@thomasfortes Form errors get displayed for the HasMany field even if I have not touched it. You can try it by creating a user and entering an age: the post field immediately shows an error.

Reply:
Don't we all love debugging forms? 😊

Turns out that the _unused_field_name wasn't being passed as a param when editing the form so Phoenix was thinking that the field was already interacted with.

I put the hidden field into the group of the checkboxes and then filtered the ids list to remove empty strings from it, there's probably other ways to do the same, but I went to the one with the minimal changes needed, let me know what you think.

Reply:
Looks good!